### PR TITLE
I will take a look

### PR DESCRIPTION
**Pull request by mgejay**
I mean, if it's senior chat, it should show the rank like adminchat... I'm not sure If this is correctly formatted.

### CODE REVIEW

**Comment by Wild1145** [comment]
By the looks of it there would be no colour formatting applied to the rank its self, something that is required.

**Comment by Wild1145** [comment]
@Scuph Any updates with this?

File: src/com/superiornetworks/icarus/modules/SeniorAdminChat.java [hunk 1]
```diff
@@ -20,7 +20,7 @@ public static void chat(CommandSender sender, String message)
             if (ICM_Rank.isRankOrHigher(p, ICM_Rank.Rank.SENIOR))
             {
                 // Some strange ass colour codes being used here...
-                p.sendMessage("§f[§bADMIN§f] §4" + sender.getName() + " §d[SrA]§f: §d " + message);
+                p.sendMessage("§f[§bADMIN§f] §4" + sender.getName() + "[" + ICM_Rank + "] "  + message);
```
Comment (Hockeyfan360) [inline src/com/superiornetworks/icarus/modules/SeniorAdminChat.java:23]: This won't even work. You used an instance of the class itself, not one of the methods that it contains.

**Comment by Hockeyfan360** [comment]
@Scuph Will not merge if the found errors cannot be fixed.

**Comment by mgejay** [comment]
I will take a look later @HockeyFan360 @Wild1145

**Comment by Hockeyfan360** [comment]
@Scuph This has been held off for quite a while. If there is inactivity for another 2-3 days, it will likely be closed.

**Pull request closed without merging**